batch/batch/driver/main.py

```diff
@@ -745,6 +746,10 @@ async def on_startup(app):
     log_store = LogStore(BATCH_BUCKET_NAME, WORKER_LOGS_BUCKET_NAME, instance_id, pool, credentials=credentials)
     app['log_store'] = log_store
 
+    zone_monitor = ZoneMonitor(app)
```

Comment:
Could we also pass `regions` and `init_zones` here, maybe after adding them to `batch_configuration`?

Reply:
Exactly, I just wasn't sure the best way to do this. Should it be in the database as a globals field in text that's JSON or should it be passed as part of the deployment.yaml as an environment variable that's also JSON? Perhaps, there's a better way than these options.

Reply:
Yet another option might be to move all the environment variables from batch/batch/batch_configuration.py into a single JSON K8s secret, similar to hail/python/hailtop/config/deploy_config.py. But that would be a larger refactoring effort.

Reply:
all the config is currently in the db, I'm inclined to stick with that unless we rethink everything. deployment env var seems natural. I don't think we have any strong case currently to change regions or initial zones at run-time.